eva-accession-import/src/test/java/uk/ac/ebi/eva/accession/dbsnp/jobs/steps/processors/SubSnpNoHgvsToVariantProcessorTest.java

```diff
@@ -0,0 +1,355 @@
+    }
+
+    @After
+    public void tearDown() throws Exception {
+    }
+
+    @Test
+    public void transformSnpForwardOrientations() throws Exception {
+        SubSnpNoHgvs subSnpNoHgvs = new SubSnpNoHgvs(25928972L, 14718243L, "A/C", ASSEMBLY, BATCH_HANDLE,
+                                                     BATCH_NAME, CHROMOSOME, CHROMOSOME_START, CONTIG_NAME,
+                                                     Orientation.FORWARD, Orientation.FORWARD, Orientation.FORWARD,
+                                                     CONTIG_START, false, false, "A", CREATED_DATE, TAXONOMY);
+
+        List<DbsnpSubmittedVariantEntity> variants = processor.process(subSnpNoHgvs);
+        // TODO: validated, match assembly and RS variant accession are being added into PR #28
+
+        DbsnpSubmittedVariantEntity expectedVariant = new DbsnpSubmittedVariantEntity(25928972L, "TODO_HASH", ASSEMBLY,
+                                                                                      TAXONOMY, PROJECT_ACCESSION,
+                                                                                      CHROMOSOME, CHROMOSOME_START, "A",
+                                                                                      "C", 14718243L, false, false,
+                                                                                      false, false, 1);
+        expectedVariant.setCreatedDate(LocalDateTime.parse("2004-06-23T21:45:00"));
+
+        // TODO: compare createdDate and hash, as they are not compared in the equals method
+
+        assertEquals(expectedVariant, variants.get(0));
+    }
+
+    @Test
+    public void transformSnpReverseSs() throws Exception {
+        SubSnpNoHgvs subSnpNoHgvs = new SubSnpNoHgvs(1984788946L, 14718243L, "T/C", ASSEMBLY, BATCH_HANDLE, BATCH_NAME,
+                                                     CHROMOSOME, CHROMOSOME_START, CONTIG_NAME, Orientation.REVERSE,
+                                                     Orientation.FORWARD, Orientation.FORWARD, CONTIG_START, false,
+                                                     false, "A", CREATED_DATE, TAXONOMY);
+
+        List<DbsnpSubmittedVariantEntity> variants = processor.process(subSnpNoHgvs);
+        DbsnpSubmittedVariantEntity expectedVariant = new DbsnpSubmittedVariantEntity(1984788946L, "TODO_HASH",
+                                                                                      ASSEMBLY, TAXONOMY,
+                                                                                      PROJECT_ACCESSION, CHROMOSOME,
+                                                                                      CHROMOSOME_START, "A", "G",
+                                                                                      14718243L, false, false, false,
+                                                                                      false, 1);
+        assertEquals(expectedVariant, variants.get(0));
```

Comment:
soft suggestion: `assertAllelesOrientation(String expectedReference, String expectedAlternate, Orientation subsnp, Orientation snp, Orientation contig, String noHgvsReference, String noHgnvsAlternate)`

would reduce the visual noise in a lot of these tests